.gitmodules

```diff
@@ -42,3 +39,6 @@
 [submodule "3rdparty/xxHash"]
 	path = 3rdparty/xxHash
 	url = https://github.com/Cyan4973/xxHash
+[submodule "3rdparty/yaml-cpp"]
+	path = 3rdparty/yaml-cpp
+	url = https://github.com/jbeder/yaml-cpp.git
```

Comment:
no need change all line, just change address url in line 24

Reply:
I think its a good idea to reorganize it. Putting 3rd party in 3rd party only makes a lot of sense.

Reply:
i have not say that, i say just put this 3 line changed on same line before 22-24 deleted, not need add news line in end, just edit already exist for less change

Reply:
I really can't see why it matters. I can't fathom anyone being confused down the line to the point where this is a maintainability issue. I wouldn't even take the time to redo it tbh.

Reply:
IMHO, it is not a good practice to edit the .gitmodules file directly. I used the git command to remove the previous submodule from Utilities and to add the new one under 3rdparty.

Reply:
@scribam you can juste edit last file and edit last commit

Reply:
He's not going to, and he shouldn't.